dev-tools/ecs-migration.yml

```diff
@@ -85,6 +85,26 @@
   alias: true
   copy_to: false
 
+- from: system.syslog.hostname
```

Comment:
Is this the hostname of the machine where the message originated?

Reply:
Yes it is